README.rst

```diff
@@ -9,6 +9,22 @@ interfaces, driver, and hardware settings.
 **This is the new upstream for python-ethtool maintained by Fedora's
 Python SIG.**
 
+Installation
+------------
+
+In order to install ``ethtool`` install its dependencies first:
+
```

Comment:
How about adding a section for installing ethtool it with the respective package managers in each distro that ethtool is packaged? (or at least Fedora and Ubuntu for starters).

Reply:
Good point, thanks. Fixed in the latest fixup commit.